pkg/reporting/client.go

```diff
@@ -32,3 +32,13 @@ type CookbookInterface interface {
 type SearchInterface interface {
 	PartialExec(idx, statement string, params map[string]interface{}) (res chef.SearchResult, err error)
 }
+
+type NodesInterface interface {
+	Get(name string) (chef.Node, error)
```

Comment:
Any reason the other interfaces return a reference and this returns an object?

Reply:
The interface was made to match the [original function](https://github.com/go-chef/chef/blob/master/node.go#L50-L54) -  in this case,  `chef.Node.Get` returns an a struct instead of a pointer.

Reply:
That bugs me every time I see it too :D